src/Storage/AbstractSessionArrayStorage.php

```diff
@@ -123,82 +116,77 @@ public function __destruct()
     /**
      * Offset Exists
      *
-     * @return bool
+     * @param non-empty-string $key
      */
     #[ReturnTypeWillChange]
-    public function offsetExists(mixed $key)
+    public function offsetExists(mixed $key): bool
     {
         return isset($_SESSION[$key]);
     }
 
     /**
      * Offset Get
      *
-     * @return mixed
+     * @param non-empty-string $key
      */
     #[ReturnTypeWillChange]
-    public function offsetGet(mixed $key)
+    public function offsetGet(mixed $key): mixed
     {
         return $_SESSION[$key] ?? null;
     }
 
     /**
      * Offset Set
      *
-     * @return void
+     * @param non-empty-string $offset
      */
     #[ReturnTypeWillChange]
-    public function offsetSet(mixed $offset, mixed $value)
+    public function offsetSet(mixed $offset, mixed $value): void
     {
         $_SESSION[$offset] = $value;
     }
 
     /**
      * Offset Unset
      *
-     * @return void
+     * @param non-empty-string $offset
      */
     #[ReturnTypeWillChange]
-    public function offsetUnset(mixed $offset)
+    public function offsetUnset(mixed $offset): void
     {
         unset($_SESSION[$offset]);
     }
 
     /**
      * Count
-     *
-     * @return int
      */
     #[ReturnTypeWillChange]
-    public function count()
+    public function count(): int
     {
         return count($_SESSION);
     }
 
     /**
-     * Seralize
-     *
-     * @return string
+     * Serialize
      */
-    public function serialize()
+    public function serialize(): string
     {
         return serialize($_SESSION);
     }
 
     /**
      * Unserialize
      *
-     * @param  string $session
-     * @return mixed
+     * @param non-empty-string $session
      */
-    public function unserialize($session)
+    public function unserialize(string $session): mixed
     {
         return unserialize($session);
     }
 
     /** @inheritDoc */
     #[ReturnTypeWillChange]
-    public function getIterator()
+    public function getIterator(): ArrayIterator
```

Comment:
There's no need to narrow this type to the exact iterator here

```suggestion
    public function getIterator(): Traversable
```